### PR TITLE
Charter

### PR DESCRIPTION
**Pull request by theheels**
Formatted the [OpenSSF Charter ](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf)into markdown

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        
+        - “Related Companies” are entities that are each a Related Company of a Member.
+- Only the legal entity which has executed a Participation Agreement and its Subsidiaries will be entitled to enjoy the rights and privileges of such OpenSSF Membership. 
+
+- If a OpenSSF Member is itself a foundation, association, consortium, open source project, membership organization, user group or other entity that has members or sponsors, then the rights and privileges granted to such OpenSSF Member will extend only to the employee-representatives of such OpenSSF Member, and not to its members or sponsors, unless otherwise approved by the Governing Board in a specific case.
+
+- OpenSSF Membership is non-transferable, non-salable and non-assignable, except an OpenSSF Member may transfer its current OpenSSF Membership benefits and obligations to a successor of substantially all of its business or assets, whether by merger, sale or otherwise; provided that the transferee agrees to be bound by this Charter and the Bylaws and policies required by LF membership.
```
Comment (TheFoxAtWork) [inline CHARTER.md:148]: Is this a good place to also added in Related Groups? Something to capture other groups already doing work in this space that the foundation can leverage and collaborate on, such as CNCF Security TAG and Kubernetes SIG-Security.  

There is some overlap of work between OpenSSF and these groups, particularly as we're driving towards the same goals (secure open source projects).  I'd like to us promote the need for collaboration.

Comment (AevaOnline) [reply CHARTER.md:148]: Charter changes require GB approval and, as far as I can tell from a quick read, this PR is just book-keeping. 

So I'd like this merged for transparency purposes first, and then we can discuss any changes/amendments. 

For what it's worth, I don't think the GB Charter is the right place to link to any external groups -- that could be a great addition to either TAC notes or specific WG pages where a WG collaborates with specific external groups.

**Review by AevaOnline — CHANGES_REQUESTED** [review]
Taking a closer look, this formatting breaks paragraph identifiers. I'll upload another PR that includes them correctly.

**Comment by lehors** [comment]
This PR should be closed. It was superseded by PR #25 and since then the charter has been made available in plain text at: https://openssf.org/about/charter/

**Comment by AevaOnline** [comment]
Closing because, as @lehors notes, the charter now has a canonical location, and does not need to be stored in git.

**Pull request closed without merging**